schedule/schedule-appengine/src/main/java/com/jasify/schedule/appengine/spi/MultipassEndpoint.java

```diff
@@ -0,0 +1,151 @@
+import com.jasify.schedule.appengine.model.consistency.InconsistentModelStateException;
+import com.jasify.schedule.appengine.model.multipass.Multipass;
+import com.jasify.schedule.appengine.spi.auth.JasifyAuthenticator;
+import com.jasify.schedule.appengine.spi.dm.JasMultipassRequest;
+import com.jasify.schedule.appengine.spi.transform.*;
+import com.jasify.schedule.appengine.util.BeanUtil;
+
+import java.util.List;
+
+import static com.jasify.schedule.appengine.spi.JasifyEndpoint.checkFound;
+import static com.jasify.schedule.appengine.spi.JasifyEndpoint.mustBeAdminOrOrgMember;
+
+/**
+ * @author wszarmach
+ * @since 09/11/15.
+ */
+@Api(name = "jasify", /* WARN: Its LAME but you have to copy & paste this section to all *Endpoint classes in this package */
+        version = "v1",
+        defaultVersion = AnnotationBoolean.TRUE,
+        description = "Jasify Schedule",
+        authenticators = {JasifyAuthenticator.class},
+        authLevel = AuthLevel.NONE,
+        transformers = {
+                /* one per line in alphabetical order to avoid merge conflicts */
+                JasAccountTransformer.class,
+                JasActivityPackageTransformer.class,
+                JasActivityTransformer.class,
+                JasActivityTypeTransformer.class,
+                JasContactMessageTransformer.class,
+                JasGroupTransformer.class,
+                JasHistoryTransformer.class,
+                JasKeyTransformer.class,
+                JasMultipassTransformer.class,
+                JasOrganizationTransformer.class,
+                JasPaymentTransformer.class,
+                JasRepeatDetailsTransformer.class,
+                JasSubscriptionTransformer.class,
+                JasTransactionTransformer.class,
+                JasUserLoginTransformer.class,
+                JasUserTransformer.class
+        },
+        auth = @ApiAuth(allowCookieAuth = AnnotationBoolean.TRUE /* todo: I don't know another way :-( */),
+        namespace = @ApiNamespace(ownerDomain = "jasify.com",
+                ownerName = "Jasify",
+                packagePath = ""))
+public class MultipassEndpoint {
+    private final MultipassDao multipassDao = new MultipassDao();
+
+    @ApiMethod(name = "multipasses.add", path = "multipasses", httpMethod = ApiMethod.HttpMethod.POST)
+    public Multipass add(@SuppressWarnings("unused") User caller, final JasMultipassRequest request) throws UnauthorizedException, ForbiddenException, BadRequestException, NotFoundException {
+        checkFound(request, "request == null");
+        final Key organizationId = checkFound(request.getOrganizationId(), "request.organizationId == null");
+        final Multipass multipass = checkFound(request.getMultipass(), "request.multipass == null");
+        mustBeAdminOrOrgMember(caller, OrgMemberChecker.createFromOrganizationId(organizationId));
+
+        try {
+            return TransactionOperator.execute(new ModelOperation<Multipass>() {
```

Comment:
I actually feel a warm and fuzzy feeling every time I see `TransactionOperator` in action :-D